### PR TITLE
Migrate enzyme Select tests to React Testing Library

### PR DESCRIPTION
**Pull request by b3h3m0th**
#### What's in this PR?

Select components unit test migration enzyme --> RTL

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js [hunk 1]
```diff
@@ -1,46 +1,63 @@
...
 
 test('The component should call the callbacks after a click', () => {
     const onClick = jest.fn();
     const afterAction = jest.fn();
-    const action = shallow(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
-    action.find('button').simulate('click');
+    render(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
+
+    const button = screen.queryByText('My action');
+    // eslint-disable-next-line testing-library/prefer-user-event
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js]: can you add why is user-event here not possible?

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
Looks like the tests are still failing.

File: src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js (hunk 1 above)
Comment (b3h3m0th) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js]: I have no idea. But when doing the same thing with `userEvent` the test just fails.

Comment (niklasnatter) [reply src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js]: is it possible that you forgot to `await` the interaction? i forgot about that a few times already 🙈

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**